src/NuGet.Core/NuGet.Build.Tasks.Console/MSBuildStaticGraphRestore.cs

```diff
@@ -534,14 +541,26 @@ private static LibraryIncludeFlags GetLibraryIncludeFlags(string value, LibraryI
         /// <param name="entryProjectPath">The full path to the main project or solution file.</param>
         /// <param name="globalProperties">An <see cref="IDictionary{String,String}" /> representing the global properties for the project.</param>
         /// <returns></returns>
-        private static List<ProjectGraphEntryPoint> GetProjectGraphEntryPoints(string entryProjectPath, IDictionary<string, string> globalProperties)
+        private List<ProjectGraphEntryPoint> GetProjectGraphEntryPoints(string entryProjectPath, IDictionary<string, string> globalProperties)
         {
             // If the project's extension is .sln, parse it as a Visual Studio solution and return the projects it contains
             if (string.Equals(Path.GetExtension(entryProjectPath), ".sln", StringComparison.OrdinalIgnoreCase))
             {
                 var solutionFile = SolutionFile.Parse(entryProjectPath);
 
-                return solutionFile.ProjectsInOrder.Where(i => i.ProjectType == SolutionProjectType.KnownToBeMSBuildFormat).Select(i => new ProjectGraphEntryPoint(i.AbsolutePath, globalProperties)).ToList();
+                IEnumerable<ProjectInSolution> projectsKnownToMSBuild = solutionFile.ProjectsInOrder.Where(i => i.ProjectType == SolutionProjectType.KnownToBeMSBuildFormat);
+                IEnumerable<ProjectInSolution> projectsNotKnownToMSBuild = solutionFile.ProjectsInOrder.Except(projectsKnownToMSBuild);
+
+                if (projectsNotKnownToMSBuild.Any())
+                {
+                    IList<string> projects = projectsNotKnownToMSBuild.Select(project => project.ProjectName).ToList();
+
+                    MSBuildLogger.LogInformation(string.Format(CultureInfo.CurrentCulture,
```

Comment:
I think this should be added at a high verbosity, such as debug. 

Information is too noisy imo.

Reply:
I agree but I am having tough time in adding tests at `Debug/Diagnostic` verbosity because as you know static graph restore defaults to "Normal" verbosity.

Reply:
hmm, so we already do the same with standard restore. 
NuGet logs everything, and MSBuild decides whether it will surface it to the user. 
I think static graph can follow the same approach.

Reply:
Consider this resolved for now.

Reply:
Please add a comment in the linked issue about this.